src/components/Calculator.js

```diff
@@ -1,69 +1,68 @@
-import React from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
 // import PropTypes from 'prop-types';
 import './calc.css';
```

Comment:
- [ ] You want to consider removing old lines of code that you are not using to follow [the JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md). 😉